kafka/assets/monitors/broker_produce_latency.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "[Kafka] High produce latency on broker",
+	"type": "metric alert",
+	"query": "avg(last_4h):avg:kafka.request.produce.time.99percentile{*} by {name} > 10000",
```

Comment:
last_4h is really long. Why such a long time-frame? Usually 5min is a good compromise

Reply:
Also we should likely use the "max" space aggregator here.